### DISCUSSION TITLE
Actual state of the package

### DISCUSSION THREAD

**Discussion by AntoineC4Ci** [BriefFiniteElementNet/BriefFiniteElement.Net#184] [Q&A]
Hello there,

I open this discussion to have some information about the actual state of the package.
- the last 2.1 release is dates from the 02/08/2022, while the last commits in sources have been made 2 weeks ago. Is there a new release to come soon ?
- The [know-issues.md](https://github.com/BriefFiniteElementNet/BriefFiniteElement.Net/blob/master/Document/know-issues.md) file mention several issues I wonder about. Indeed, this file is also 2 year old now, so is it outdated ? More specific : are the concentrated loads fixed ?
- the documentation is very scattered, and a lot of it is outdated, links are sometimes broken. Is there any public project to reorganize this ?

Regards

**Comment by epsi1on** [comment]
happy to hear this is important to you :)
- each several month nuget is published. no strict schedule. definitely there will be a publish in future not sure about the time
- most items are resolved of that file
- could you please describe more? and yes, if there are contributors then definitely doc have room to improve.

**Reply by AntoineC4Ci** [reply]
Thank you for the quick answer.
If I have to use this package, I'll do what I can to clean up the documentation.

**Reply by epsi1on** [reply]
good to hear that, please feel free to make pull requests

**Comment by epsi1on** [comment]
inactivity
